Texstyle/Texstyle/Array+Text.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright © 2019 Rosberry. All rights reserved.
+//
+
+public extension Array where Element: Text {
+
+    func joined(separatorText: Text = Text(value: "", style: .init())) -> Text {
```

Comment:
These default values aren't usable IMO. It would be better to make these parameters nullable and if they are use style of the first element in each pair. For example array has 3 elements and parameters are `nil` so texts would be combined like this
`[0] + ""(style of [0]) + [1] + ""(style of [1]) + [2]`

Otherwise simple call of `.joined()` would be completely unusable

Reply:
I like the idea about the optional separator. But not sure about styles. Let's discuss it.